sdk/tables/Azure.Data.Tables/tests/TableClientLiveTests.cs

```diff
@@ -116,6 +116,63 @@ public void ValidateSasCredentials()
             Assert.That(async () => await sasTableclient.UpsertEntityAsync(CreateTableEntities("partition", 1).First(), TableUpdateMode.Replace).ConfigureAwait(false), Throws.InstanceOf<RequestFailedException>().And.Property("Status").EqualTo((int)HttpStatusCode.Forbidden));
         }
 
+        /// <summary>
+        /// Validates the functionality of the TableClient.
+        /// </summary>
+        [RecordedTest]
+        public async Task ValidateSasCredentialsWithRowKeyAndPartitionKeyRanges()
+        {
+            List<TableEntity> entitiesToCreate = CreateTableEntities(PartitionKeyValue, 2);
+
+            // Create a SharedKeyCredential that we can use to sign the SAS token
+            var credential = new TableSharedKeyCredential(AccountName, AccountKey);
+
+            // Build a shared access signature with only All permissions.
+            TableSasBuilder sas = client.GetSasBuilder(TableSasPermissions.All, new DateTime(2040, 1, 1, 1, 1, 0, DateTimeKind.Utc));
```

Comment:
nit:  Should this be using `DateTimeOffset`?

Reply:
No, it doesn't really matter here.